src/Sfx-Standalone/main.ts

```diff
@@ -22,38 +19,23 @@ async function startup(): Promise<void> {
     }
 
     log.writeInfoAsync("Starting up connect-cluster prompt.");
-    const prompt_connectCluster = await sfxModuleManager.getComponentAsync("prompt.connect-cluster");
-    const clusterUrl = await prompt_connectCluster.openAsync();
-
-    if (clusterUrl) {
-        // Start up the main window.
-        global["TargetClusterUrl"] = clusterUrl;
-        const mainWindow = await sfxModuleManager.getComponentAsync("electron.browser-window", null);
-
-        mainWindow.setMenuBarVisibility(false);
-
-        log.writeEventAsync("connect-cluster", { "clusterId": uuidv5(clusterUrl, uuidv5.URL) });
-        mainWindow.loadURL(resolve("sfx/index.html"));
-    } else {
-        log.writeInfoAsync("No cluster url provided.");
-        log.writeInfoAsync("app.quit().");
-
-        app.quit();
-        return;
-    }
-
-    // Trigger update activity.
-    (await sfxModuleManager.getComponentAsync("update.service")).updateAsync();
+    const mainWindow = await sfxModuleManager.getComponentAsync("sfx.main-window");
+    await mainWindow.loadAsync();
 
     // Handle "window-all-closed" event.
     app.removeAllListeners("window-all-closed");
     app.once("window-all-closed", async () => {
-        const log = await sfxModuleManager.getComponentAsync("logging.default");

```

Comment:
This is trying not to hold log object inside the callback which will create a closure. Obtaining the log object when it's necessary.